packages/esm-service-queues-app/src/queue-table/queue-table-by-status.component.tsx

```diff
@@ -0,0 +1,103 @@
+import { InlineNotification, Tab, TabList, TabPanel, TabPanels, Tabs } from '@carbon/react';
+import { isDesktop, useLayoutType } from '@openmrs/esm-framework';
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import styles from '../active-visits/active-visits-table.scss';
+import { useQueueEntries } from '../hooks/useQueueEntries';
+import { type Concept, type Queue, type QueueTableColumn, type QueueTableTabConfig } from '../types';
+import QueueTableNameCell from './cells/queue-table-name-cell.component';
+import { QueueTable } from './queue-table.component';
+import QueueTableStatusCell from './cells/queue-table-status-cell.component';
+import QueueTablePriorityCell from './cells/queue-table-priority-cell.component';
+import { useNavigate } from 'react-router-dom';
+
+interface QueueTableByStatusProps {
+  queue: Queue;
+  status: Concept;
+  configByStatus?: Map<string, QueueTableTabConfig>;
+  all?: QueueTableTabConfig;
+}
+
+const DefaultQueueTableColumns: QueueTableColumn[] = [
+  {
+    headerI18nKey: 'name',
+    cellComponent: (queueEntry) => <QueueTableNameCell queueEntry={queueEntry} />,
+  },
+  {
+    headerI18nKey: 'priority',
+    cellComponent: (queueEntry) => <QueueTablePriorityCell queueEntry={queueEntry} />,
+  },
+  {
+    headerI18nKey: 'status',
+    cellComponent: (queueEntry) => <QueueTableStatusCell queueEntry={queueEntry} />,
+  },
```

Comment:
Can we add more column components in to achieve feature parity with the existing queue table rendered in the active visits table / service queues page, and then add some more of these into the default view?  We can leave off `QueueNumber`, as this is a custom column for only some implementations, but if we can add in `Coming From` and `Wait time (mins)`, those would be useful. In a later PR we can add support for actions.